17.md

```diff
@@ -0,0 +1,34 @@
+NIP-17
+======
+
+Sensitive Content / Content Warning
+-----------------------------------
+
+`draft` `optional` `author:fernandolguevara`
+
+Like in mastodon, `content-warning` tag enables users to specify if the event's content needs to be approved by readers to be shown.
+Clients can show a blurred section in the content place.
```

Comment:
"Clients can hide the content until the user acts on it"

No need to imply a blur, clients will deal with this how they want